### PR TITLE
chore(core): create/drop/truncate table query registry fix

### PR DESCRIPTION
**Pull request by glasstiger**
- fixes an issue where CREATE TABLE statements are getting stuck in query registry
 - also makes sure that DROP TABLE and TRUNCATE TABLE gets registered
 - and fixes CREATE TABLE LIKE which ignored the OWNED BY clause

required by https://github.com/questdb/questdb-enterprise/pull/523

### CODE REVIEW

**Comment by glasstiger** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 18 / 21 (85.71%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/SqlCompilerImpl.java|18|21|85.71%|

**Comment by glasstiger** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 24 / 29 (82.76%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/SqlParser.java|6|8|75.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCompilerImpl.java|18|21|85.71%|

**Comment by glasstiger** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 24 / 29 (82.76%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/SqlParser.java|6|8|75.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCompilerImpl.java|18|21|85.71%|

File: core/src/main/java/io/questdb/griffin/engine/ops/CreateTableOperationFuture.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package io.questdb.griffin.engine.ops;
```
Comment (ideoma) [inline core/src/main/java/io/questdb/griffin/engine/ops/CreateTableOperationFuture.java:1]: nit: no header

**Comment by glasstiger** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 457 / 484 (94.42%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/SqlParserCallback.java|1|3|33.33%|
|:large_blue_circle:|io/questdb/griffin/SqlParser.java|7|9|77.78%|
|:large_blue_circle:|io/questdb/griffin/SqlCompilerImpl.java|18|21|85.71%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/CreateTableOperationImpl.java|223|241|92.53%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/CreateTableOperationBuilderImpl.java|205|207|99.03%|
|:large_blue_circle:|io/questdb/griffin/QueryBuilder.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/CreateTableOperationFuture.java|2|2|100.00%|

**Comment by glasstiger** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 458 / 484 (94.63%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/SqlParserCallback.java|1|3|33.33%|
|:large_blue_circle:|io/questdb/griffin/SqlParser.java|7|9|77.78%|
|:large_blue_circle:|io/questdb/griffin/SqlCompilerImpl.java|18|21|85.71%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/CreateTableOperationImpl.java|223|241|92.53%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/CreateTableOperationBuilderImpl.java|206|207|99.52%|
|:large_blue_circle:|io/questdb/griffin/QueryBuilder.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/CreateTableOperationFuture.java|2|2|100.00%|

**Comment by glasstiger** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 458 / 485 (94.43%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/SqlParserCallback.java|1|3|33.33%|
|:large_blue_circle:|io/questdb/griffin/SqlParser.java|7|9|77.78%|
|:large_blue_circle:|io/questdb/griffin/SqlCompilerImpl.java|18|21|85.71%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/CreateTableOperationImpl.java|223|241|92.53%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/CreateTableOperationBuilderImpl.java|205|207|99.03%|
|:large_blue_circle:|io/questdb/griffin/QueryBuilder.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/CreateTableOperationFuture.java|2|2|100.00%|

**Review by ideoma — APPROVED** [review]

**Comment by glasstiger** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 458 / 485 (94.43%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/SqlParserCallback.java|1|3|33.33%|
|:large_blue_circle:|io/questdb/griffin/SqlParser.java|7|9|77.78%|
|:large_blue_circle:|io/questdb/griffin/SqlCompilerImpl.java|18|21|85.71%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/CreateTableOperationImpl.java|223|241|92.53%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/CreateTableOperationBuilderImpl.java|205|207|99.03%|
|:large_blue_circle:|io/questdb/griffin/QueryBuilder.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/CreateTableOperationFuture.java|2|2|100.00%|

**Pull request merged**